rust-lib/src/lib.rs

```diff
@@ -358,21 +359,26 @@ pub extern "C" fn transfer(
 
 fn _accept(
     runtime: &COpaqueStruct,
-    consignment_bytes: *mut c_char,
+    consignment_bytes: *const u8,
+    consignment_length: c_int,
     reveal_outpoints: *mut c_char,
```

Comment:
Cool. Just noticed that we have `mut`s everywhere, while we do not need them. Can we change that part as well?